### ISSUE TITLE
Qwen image edit 2509  sigkill

### ISSUE THREAD

**Issue by maxroccobg-cyber** [ostris/ai-toolkit#993]
Hello Ostris Team,
I am trying to train a LoRA for Qwen-Image-Edit-2509, but every training attempt fails before the first training step.
I have already spent several hours testing different configurations and GPUs, so I would like to know if I am doing something wrong or if this is a known issue.
Model
Model: Qwen/Qwen-Image-Edit-2509
Architecture: qwen_image_edit_plus
Dataset
140 images
Every image has its own .txt caption
Trigger word: Alice
Dataset uploads successfully without any errors.
Training Configuration
Rank: 32
Batch Size: 1
Steps: 3500
Learning Rate: 0.0001
Optimizer: AdamW8Bit
Cache Latents: Enabled
Compile Model: Disabled
Resolutions: 512 and 768 only (1024 disabled)
qfloat8 for both Transformer and Text Encoder
I also tested all of the following:
Low VRAM ON
Low VRAM OFF
Layer Offloading ON
Layer Offloading OFF
Transformer Offload 100%
Text Encoder Offload 100%
The result is always the same.
GPUs Tested
RTX 4090 (24 GB)
Training always stops with:
CUDA Out of Memory
This is understandable because of VRAM limitations.
RTX A6000 (48 GB)
The model downloads successfully.
The transformer loads successfully.
The tokenizer loads successfully.
Everything appears to initialize correctly.
However, before the first training step begins, the process is terminated with:
Job process terminated with signal SIGKILL
The training never reaches Step 1.
My Goal
My goal is to create an identity LoRA of one person using approximately 140 captioned images, so that I can use this LoRA later inside the Qwen-Image-Edit-2509 ComfyUI workflow.
My Questions
Is LoRA training for Qwen-Image-Edit-2509 currently fully supported?
Is a standard image + caption dataset sufficient for identity LoRA training, or does qwen_image_edit_plus require a different dataset structure (for example edit pairs or control images)?
Is the SIGKILL error a known issue with the current Ostris deployment?
Is there any recommended configuration for training identity LoRAs with Qwen-Image-Edit-2509?
If my dataset format is incorrect, could you please explain the expected dataset structure?
I have attached screenshots of the errors, the configuration, and the logs.
Thank you very much for your help.

<img width="1440" height="1080" alt="Image" src="https://github.com/user-attachments/assets/a11ea8dd-600d-4623-bcad-f7b857f6ef53" />

<img width="1440" height="1080" alt="Image" src="https://github.com/user-attachments/assets/82deb71b-48ef-493f-8a31-9cd85bca73ec" />

**Comment by brand175** [comment]
@maxroccobg-cyber You need to set the layer offloading to TE to 0% to offload the TE or it wont work, since the 4090 and RTX6000 doesn't have enough VRAM to have both the model and TE. You also need 64GB of RAM for offloading, Qwen uses too much resources. I'd recommend using Krea2 instead. You need to disable sampling, not having enough VRAM will cause generating samples to OOM the training (At this point, you would risk OOM during training). For your RTX6000, I think you need to install the latest CUDA 13 drivers which include tools to compile wheels, which requires microsoft VS tools to install correctly. IIRC the nvidia app has an option to install the basic drivers for PyTorch when you install the graphics driver.
